content/en/getting-started/quick-start.md

```diff
@@ -179,7 +179,7 @@ It is simple. Just call:
 hugo
 ```
 
-Output will be in `./public/` directory by default (`-d`/`--destination` flag to change it, or set `publishdir` in the config file).
+Output will be in `./private/` directory by default (`-d`/`--destination` flag to change it, or set `publishdir` in the config file).
```

Comment:
Why `private`?

Reply:
I suspect this is another bogus Hacktober "free t-shirt" PR.

Reply:
I see.